pyiss/__init__.py

```diff
@@ -77,17 +77,17 @@ def pass_times(self, latitude, longitude, altitude=None, number=None):
         # Build request
         payload = {'lat': latitude, 'lon': longitude}
 
-        if altitude is not None:
+        if altitude != None:
             payload['alt'] = altitude
 
-        if number is not None:
+        if number != None:
```

Comment:
These ones were actually correct, `None` is a singleton and is idiomatically compared using `is`/`is not`.